<ui version="4.0" >
 <class>ResistorCalibrateForm</class>
 <widget class="QDialog" name="ResistorCalibrateForm" >
  <property name="windowModality" >
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>182</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>калибровка резистора</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QPushButton" name="pbStart" >
       <property name="text" >
        <string>Старт</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbStop" >
       <property name="text" >
        <string>Стоп</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbExit" >
       <property name="text" >
        <string>Выход</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="frame" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="label" >
          <property name="text" >
           <string>&lt;b>Резистор</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="sbResistor" >
          <property name="maximum" >
           <number>255</number>
          </property>
          <property name="value" >
           <number>127</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QSlider" name="hsResistor" >
        <property name="maximum" >
         <number>255</number>
        </property>
        <property name="value" >
         <number>127</number>
        </property>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_2" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="label_2" >
          <property name="text" >
           <string>&lt;b>Ноль</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="lcdNumber" >
          <property name="frameShape" >
           <enum>QFrame::Box</enum>
          </property>
          <property name="frameShadow" >
           <enum>QFrame::Plain</enum>
          </property>
          <property name="smallDecimalPoint" >
           <bool>false</bool>
          </property>
          <property name="numDigits" >
           <number>4</number>
          </property>
          <property name="segmentStyle" >
           <enum>QLCDNumber::Flat</enum>
          </property>
          <property name="intValue" stdset="0" >
           <number>0</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>hsResistor</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sbResistor</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>135</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel" >
     <x>118</x>
     <y>43</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sbResistor</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hsResistor</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>118</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel" >
     <x>135</x>
     <y>70</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbExit</sender>
   <signal>clicked()</signal>
   <receiver>ResistorCalibrateForm</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel" >
     <x>199</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ResistorCalibrateForm</sender>
   <signal>rejected()</signal>
   <receiver>pbStop</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>199</x>
     <y>211</y>
    </hint>
    <hint type="destinationlabel" >
     <x>163</x>
     <y>49</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
